src/selectedtests/task_mappings/mappings.py

```diff
@@ -54,6 +55,13 @@ def create_task_mappings(
         :param build_regex: Regex pattern to match build variant names against. Defaults to "!.*"
         :return: An instance of the task mappings class
         """
+        log = LOGGER.bind(
```

Comment:
Since this repo isn't using cached logging yet, that hasn't come up yet, but I have created an issue so that we add cached logging and use `tmp_bind`: https://jira.mongodb.org/browse/TIG-2149

Reply:
Since this repo isn't using cached logging yet, that hasn't come up yet, but I have created an issue so that we add cached logging and use `tmp_bind`: https://jira.mongodb.org/browse/TIG-2149